pkg/users/delete-group-dialog.js

```diff
@@ -54,7 +54,8 @@ export function delete_group_dialog(group) {
                         prog.push("-f");
                     prog.push(group.name);
 
-                    return cockpit.spawn(prog, { superuser: "require", err: "message" });
+                    return cockpit.spawn(prog, { superuser: "require", err: "message" })
+                            .then(() => cockpit.location.go("/"));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230118-134524-16099183-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)